build.gradle

```diff
@@ -174,20 +174,28 @@ test {
         exceptionFormat = 'full'
     }
 }
-downloadRobotServerPlugin {
-    version.set(remoteRobotVersion)
-}
```

Comment:
I removed `downloadRobotServerPlugin` based on the doc https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-migration.html#downloadrobotserverplugin for migrating from gradle intellij plugin version `1.*`

<img width="822" alt="Screenshot 2024-11-18 at 9 45 54 AM" src="https://github.com/user-attachments/assets/11d59507-2bdf-49b4-91aa-894cb333a754">

I have added `plugins { robotServerPlugin() }` in the dependency helper for custom task `runIdeForUiTests`.